src/components/AboutPage/AboutPage.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { forwardRef } from 'react';
+
+interface AboutPageProps {
+  // You can add any props that the AboutPage component might need in the future
+}
+const AboutPage = forwardRef<HTMLDivElement, AboutPageProps>((props, ref) => {
+  return (
+    <div ref={ref} id="events-section" className=" h-screen">
```

Comment:
@Hariharan482 id used here is events-section but it is about page

Reply:
changed it to about-section